### PR TITLE
Fix `min_input_args` api_mapping data PART.2

### PR DESCRIPTION
**Pull request by RedContritio**
### PR Docs

添加了部分映射数据的 `min_input_args`

### PR APIs

```bash

```

### CODE REVIEW

**Review by zhwesky2010 — COMMENTED** [review]
min_input_args是不包含**指定关键字**的最小输入，注意仔细配置具体值

**Comment by RedContritio** [comment]
> min_input_args是不包含**指定关键字**的最小输入，注意仔细配置具体值

该 PR 已移除对 `min_input_args` 转换过程中检查的提交

**Review by zhwesky2010 — COMMENTED** [review]
注意min_input_args的定义，看另一个PR

File: paconvert/api_mapping.json [hunk 1]
```diff
@@ -10591,6 +10617,7 @@
   "torch.nn.functional.leaky_relu": {
     "Matcher": "GenericMatcher",
     "paddle_api": "paddle.nn.functional.leaky_relu",
+    "min_input_args": 0,
```
[the remark this answers is not in the document]
Comment (RedContritio) [reply paconvert/api_mapping.json:10620]: 已更新

**Comment by zhwesky2010** [comment]
冲突了，需要处理下

File: paconvert/api_mapping.json [hunk 2]
```diff
@@ -9587,6 +9606,7 @@
   "torch.nn.Softmax": {
     "Matcher": "SoftmaxMatcher",
     "paddle_api": "paddle.nn.Softmax",
+    "min_input_args": 1,
```
Comment (zhwesky2010) [inline paconvert/api_mapping.json:9609]: 这个dim是必选的吗

`torch.nn.Softmax(dim=None)`

https://pytorch.org/docs/stable/generated/torch.nn.Softmax.html#softmax

File: tests/test_sparse_csr_tensor.py [hunk 3]
```diff
@@ -94,3 +94,20 @@ def test_case_5():
         """
     )
     obj.run(pytorch_code, ["result"])
+
+
+# paddle not support convert without shape
```
Comment (zhwesky2010) [inline tests/test_sparse_csr_tensor.py:99]: 这个原因是，torch可以不指定shape，paddle必须指定shape？

File: paconvert/api_mapping.json (hunk 2 above)
Comment (RedContritio) [reply paconvert/api_mapping.json:9665]: 不是，已取消，之前标注是因为如果不传入 dim，会导致转换结果不同。

**Comment by RedContritio** [comment]
> 冲突了，需要处理下

已解决

**Review by zhwesky2010 — APPROVED** [review]
